tests/test_database_manager.py

```diff
@@ -0,0 +1,171 @@
+"""
+test_database_manager.py - UT for the databaseManager
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+This module is built on top of the Pydle system.
+"""
+import pytest
+import pyodbc
+from Modules.database_manager import DatabaseManager
+
+
+@pytest.fixture
+def prepare_dbm(dbm_fx: DatabaseManager):
+    """
+    Drops all the tables
+    """
+    if dbm_fx.marker is not None:
+        return
+    dbm_fx.marker = False
+    table_names = ("testtablehas",
+                   "testtableselect",
+                   "testtablecreate",
+                   "testtableinsert",
+                   "testtableupdate",
+                   "testtabledelete",
+                   )
+    for name in table_names:
+        try:
+            dbm_fx.cursor.execute(f"DROP TABLE {name};")
+        except (pyodbc.ProgrammingError, pyodbc.OperationalError, pyodbc.DataError):
+            pass
+
+
+# noinspection PyProtectedMember
+@pytest.mark.asyncio
+@pytest.mark.database
+@pytest.mark.usefixtures("prepare_dbm")
+class TestStuff(object):
+
+    @pytest.mark.skip(reason="Not a test")
+    def finalize(self, dbm_fx):

```

Comment:
skip is not needed, as pytest won't detect it as a test anyways.
pytest recognizes anything beginning with `test_` as a test by default

Reply:
you are right, this was a leftover.